recipes/skprogs-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "skprogs" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}-python
```

Comment:
We never formalized this and the change below is more os a suggestion but most of the python wrappers use a prefix instead of a suffix. 

```suggestion
  name: python-{{ name|lower }}
```

Reply:
Thanks for the review, I guess we would like to keep this recipe consistent with e.g. the [dftbplus-python-feedstock](https://github.com/conda-forge/dftbplus-python-feedstock). If it is really just a loose convention, I would prefer to keep the current name.